### PR TITLE
Autogenerate common Istio annotations

### PR DESCRIPTION
**Pull request by nmittler**

### CODE REVIEW

File: annotation/annotations.gen.go [hunk 1]
```diff
@@ -0,0 +1,285 @@
+// Copyright 2019 Istio Authors
```
Comment (geeknoid) [inline annotation/annotations.gen.go]: No need for a copyright on a generated file.

File: annotation/annotations.yaml [hunk 2]
```diff
@@ -0,0 +1,136 @@
+annotations:
```
Comment (geeknoid) [inline annotation/annotations.yaml]: This one could use a copyright.

File: annotation/annotations.gen.go [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+//
+// nolint: lll
+//go:generate go run $GOPATH/src/istio.io/api/annotation/cmd/gen/main.go --input=$GOPATH/src/istio.io/api/annotation/annotations.yaml --output=$GOPATH/src/istio.io/api/annotation/annotations.gen.go
+//go:generate goimports -w annotations.gen.go
+
+package annotation
+
+var (
+	IngressClass = Instance{
+		Name: "kubernetes.io/ingress.class",
```
Comment (nmittler) [inline annotation/annotations.gen.go]: @geeknoid I suspect I should remove non-Istio annotations (this is the only one) ... wdyt?

Comment (nmittler) [reply annotation/annotations.gen.go]: Decided to just remove it.

**Pull request merged**